docs/book/src/troubleshooting.md

```diff
@@ -74,3 +74,9 @@ than 4MiB by specifying the `--max-call-recv-msg-size=<size in bytes>` argument
 
 Note that this may also increase memory resource consumption of the `secrets-store` container, so you should also
 consider increasing the memory limit as well.
+
+### failed to get CSI client: `driver name secrets-store.csi.k8s.io not found in the list of registered CSI drivers`
+### Volume mount fails with `"GRPC error" err="failed to mount objects, error: failed to write file: no such file or directory`
+Some Kubernetes distros use a custom `kubeletRootDir` path. This may cause errors such as volume mount failures or failures
```

Comment:
```suggestion
Some Kubernetes distros (such as Rancher and Microk8s) use a custom `kubeletRootDir` path. This may cause errors such as volume mount failures or failures
```

Reply:
should we add known paths too? for rancher, it looks like its `/opt/rke/var/lib/kubelet` and for microk8s, it's `/var/snap/microk8s/common/var/lib/kubelet`

Reply:
> should we add known paths too? for rancher, it looks like its /opt/rke/var/lib/kubelet and for microk8s, it's /var/snap/microk8s/common/var/lib/kubelet

If we're adding these _known_ defaults, let's add a link to the docs/gh issues where this was documented. If it changes the future, there will be an external reference for the user to follow-up on.

Reply:
agreed. however, these are not documented. at least i couldn't find any "official" references